packages/astro/src/types/public/config.ts

```diff
@@ -1328,58 +1328,62 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 * - `fixed` - The image will maintain its original dimensions.
 		 * - `full-width` - The image will scale to fit the container, maintaining its aspect ratio.
 		 */
-		experimentalLayout?: ImageLayout | undefined;
+		layout?: ImageLayout | undefined;
 		/**
 		 * @docs
-		 * @name image.experimentalObjectFit
+		 * @name image.objectFit
 		 * @type {ImageFit}
 		 * @default `"cover"`
+		 * @version 5.10.0
 		 * @description
 		 * The default object-fit value for responsive images. Can be overridden by the `fit` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
+		 * Requires a value for `layout` to be set.
 		 */
-		experimentalObjectFit?: ImageFit;
+		objectFit?: ImageFit;
 		/**
 		 * @docs
-		 * @name image.experimentalObjectPosition
+		 * @name image.objectPosition
 		 * @type {string}
 		 * @default `"center"`
+		 * @version 5.10.0
 		 * @description
 		 * The default object-position value for responsive images. Can be overridden by the `position` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
+		 * Requires a value for `layout` to be set.
 		 */
-		experimentalObjectPosition?: string;
+		objectPosition?: string;
 		/**
 		 * @docs
-		 * @name image.experimentalBreakpoints
+		 * @name image.breakpoints
 		 * @type {number[]}
 		 * @default `[640, 750, 828, 1080, 1280, 1668, 2048, 2560] | [640, 750, 828, 960, 1080, 1280, 1668, 1920, 2048, 2560, 3200, 3840, 4480, 5120, 6016]`
+		 * @version 5.10.0
 		 * @description
-		 * The breakpoints used to generate responsive images. Requires the `experimental.responsiveImages` flag to be enabled. The full list is not normally used,
+		 * The breakpoints used to generate responsive images. Requires a value for `layout` to be set. The full list is not normally used,
 		 * but is filtered according to the source and output size. The defaults used depend on whether a local or remote image service is used. For remote services
 		 * the more comprehensive list is used, because only the required sizes are generated. For local services, the list is shorter to reduce the number of images generated.
 		 */
-		experimentalBreakpoints?: number[];
+		breakpoints?: number[];
 		/**
 		 * @docs
-		 * @name image.experimentalDefaultStyles
+		 * @name image.responsiveStyles
 		 * @type {boolean}
-		 * @default `true`
+		 * @default `false`
+		 * @version 5.10.0
 		 * @description
-		 * Whether to automatically add global styles to ensure that experimental images resize correctly. This is enabled by default, but can be disabled if you want to manage the styles yourself.
-		 * This option is only used when the `experimental.responsiveImages` flag is enabled.
+		 * Whether to automatically add global styles to ensure that responsive images resize correctly.
+		 * This option is only used when `layout` is set to `constrained`, `full-width`, or `fixed` using the configuration or the `layout` prop on the image component.
```

Comment:
```suggestion
		 * Whether to automatically add global styles for responsive images. We recommend enabling this option to ensure that images resize correctly.
		 * This option is only used when `layout` is set to `constrained`, `full-width`, or `fixed` using the configuration or the `layout` prop on the image component.
		 * 
		 * See https://docs.astro.build/en/guides/images/#responsive-image-styles for more information.
```

@sarah11918 how about this